taxi/urls.py

```diff
@@ -6,7 +6,7 @@
     CarDetailView,
     DriverListView,
     DriverDetailView,
-    ManufacturerListView,
+    ManufacturerListView
```

Comment:
It is better to place `,` at the end of the line in the future so that git would show more meaningful changes when new views would be added

Reply:
Ok , thank)